src/rethinkdb/utils.cljc

```diff
@@ -14,11 +14,7 @@
 #?(:clj (defn str->bytes
           "Creates a ByteBuffer of size n bytes containing string s converted to bytes"
           [^String s]
-          (let [n (count s)
-                buf (ByteBuffer/allocate n)]
-            (doto buf
-              (.put (.getBytes s)))
-            (.array buf))))
+          (.getBytes s "UTF-8")))
```

Comment:
Would need to check the code to see, but the comment should be cleaned up at least.